lib/client.js

```diff
@@ -3,6 +3,7 @@
 // Util Modules
 const EventEmitter      = require('events').EventEmitter;
 const debug             = require('debug')('bacstack');
+debug.trace             = require('debug')('bacstack:trace');
```

Comment:
This shouldn't be a global variable. Please stick to `debug`.

Reply:
I wanted to have two different types of debugging messages, because the `trace` ones are very verbose.  By doing it this way you can choose to hide the `bacstack:trace` messages, so you only see the more important `bacstack` debug messages.  For example, like this:

    DEBUG='*,-bacstack:trace' node index.js

If these are all changed to be only `bacstack` then it becomes difficult to see important messages as they are lost amongst the trace.

In my environment the `trace` messages are extremely useful to ensure firewall rules are working and when debugging why messages are being ignored, but once the code is working there are pages of them every few seconds so they need to be disabled in order to view the normal debug messages.

I am happy to do this another way, but I think it is very important to have the two different levels of messages so that one can be disabled while leaving the other active.